### PR TITLE
add i2p related addresses and muwire.com

### PR DESCRIPTION
**Pull request by xhdix**

### CODE REVIEW

File: lists/global.csv [hunk 1]
```diff
@@ -1435,3 +1435,7 @@ https://149.112.112.9:5053/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQA
 https://9.9.9.9:5053/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HOST,Hosting and Blogging Platforms,2020-07-16,Chelchela,
 https://doh.dns.sb/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HOST,Hosting and Blogging Platforms,2020-07-16,Chelchela,
 https://mozilla.cloudflare-dns.com/dns-query?dns=q80BAAABAAAAAAAAA3d3dwdleGFtcGxlA2NvbQAAAQAB,HOST,Hosting and Blogging Platforms,2020-07-16,Chelchela,
+https://i2p.net/,ANON,Anonymization and circumvention tools,2020-07-30,Chelchela,
```
Comment (hellais) [inline lists/global.csv:1438]: What is the difference between this site and https://geti2p.net/en/?

Comment (xhdix) [reply lists/global.csv:1438]: This is just an outdated edition. And it does not seem to be part of the main site. It seems better to delete it. (?)
image
image

**Comment by xhdix** [comment]
conflicts fixed.

**Comment by hellais** [comment]
LGTM

**Pull request merged**